shell/platform/android/io/flutter/embedding/android/FlutterSurfaceView.java

```diff
@@ -46,6 +47,8 @@ public void surfaceCreated(SurfaceHolder holder) {
       Log.d(TAG, "SurfaceHolder.Callback.surfaceCreated()");
       isSurfaceAvailableForRendering = true;
 
+      holder.setFormat(PixelFormat.RGBA_8888);
```

Comment:
Seems like this should be in a separate PR since it's unrelated to the other changes.

Reply:
Removed.